src/style.css

```diff
@@ -0,0 +1,68 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: bold;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+  font-family: Avenir, sans-serif;
+}
+
+.search {
+  margin: 20px 8px;
+}
+
+.search__input {
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+  box-sizing: border-box;
+  border-radius: 4px;
+  width: 100%;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  box-shadow: 0 1px 8px 0 #e1e7ed;
+  border: 1px solid rgba(225, 231, 237, 1);
+}
+
+.search__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search__input:focus {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+  color: #3d4e61;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
+}
+
+.search__input::placeholder {
+  color: #3d4e61;
+}
+
+.search__input--big {
```

Comment:
Actually, you can make one of them the default one, so another one will be with modifier